README.md

```diff
@@ -2,727 +2,369 @@
+[{'target_id': u'0240000032044e4500257009997b00386781000097969900', ... 'platform_name': u'CUSTOM_PLATFORM'}]
+>>> mbeds.mock_manufacture_id('0240', '', oper='-')
+>>> mbeds.list_mbeds()
+[{'target_id': u'0240000032044e4500257009997b00386781000097969900', ... 'platform_name': u'K64F'}]
 ```
 
-Let's say we want change ```serial_port```'s value to other COM port. For example we are using other serial port (e.g. while debugging) on our device as standard output.
-To do so we would have to create a new file called ```mbedls.json``` in directory where want to use this modification. File content could look like this: a JSON file where keys are ```target_id```'s and values are dictionaries with new values:
+#### Arguments
 
-```
-$ cat mbedls.json
-{
-    "0240022648cb1e77000000000000000000000000b512e3cf" : {
-        "serial_port" : "MyComPort01"
-    }
-}
-```
+##### `mid`
 
-Now, when we issue ```mbedls``` command in this directory our auto-detection data will be replaced:
-```
-$ mbedls
-+--------------+---------------------+------------+------------+-------------------------------------------------+
-|platform_name |platform_name_unique |mount_point |serial_port |target_id                                        |
-+--------------+---------------------+------------+------------+-------------------------------------------------+
-|K64F          |K64F[0]              |F:          |MyComPort01 |0240022648cb1e77000000000000000000000000b512e3cf |
-+--------------+---------------------+------------+------------+-------------------------------------------------+
-```
+**Required**
 
-# Mocking new or existing target to custom platform name
-Command line switch ```--mock``` provide simple manufacturers ID masking with new platform name.
-Users should be able to add temporarily new ```MID``` -> ```platform_name``` mapping when e.g. prototyping.
+The first four characters of the TargetID that you want to mock.
 
-Mock configuration will be stored in `$HOME/.mbed-ls/` directory, in local file ```.mbedls-mock```.
+##### `platform_name`
 
-**Note***: ```MID``` stands for "manufacturers ID". `MID` is first four (4) characters of ```target_id``` string. Example: If ```target_id``` is ```02400221A0811E505D5FE3E8```, corresponding manufacturers ID is ```0240```.
+**Required**
 
-## Mock command line examples
-* Mock command line parameter: `--mock` or (switch `-m`)
-* Add new / mask existing mapping ```MID``` -> ```platform_name``` and assign `MID`:
-    * ```$ mbedls --mock MID:PLATFORM_NAME``` or
-    * ```$ mbedls --mock MID1:PLATFORM_NAME1,MID2:PLATFORM_NAME2```
-    * Example: `$ mbedls --mock 0818:NUCLEO_F767ZI`
-* Remove masking with '!' prefix: `$ mbedls --mock !MID`
-* Remove all maskings using !* notation: `$ mbedls --mock !*`
-* Combine above using comma (`,`) separator: `$ mbedls --mock MID1:PLATFORM_NAME1,!MID2`
+The name of the platform that is returned for any platform that has a `target_id` that matches the first four characters specified in `mid`.
```

Comment:
Query: Could we rewrite this for clarity? Is it saying that `mid` specifies characters, and that any platform with a `target_id` that matches the first four characters that `mid` specifies returns its own name? Or it returns the name of a different platform?